pages/design/composed-template-empty.md

```diff
@@ -0,0 +1,85 @@
+---
+name:
+rule_type: composed
+description: |
+  This rule checks ...
+
+success_criterion:
+- x.y.z
+
+atomic_rules:
+- 
+
+authors:
+-
+---
+
+## Aggregation Definition
+
+### Applicability
+
+The rule applies to any (??) element ...
+
+### Expectation
+
+For each test target, the outcome of << Logic here >>:
```

Comment:
"For each test target, the outcome of << Logic here >>:" 

--> 

"For each test target, the outcome of (at least one of / all of / any of etc.) the following rules is (passed / failed / etc.):"